client/modules/IDE/reducers/files.js

```diff
@@ -31,6 +31,9 @@ const defaultCSS =
   margin: 0;
   padding: 0;
 }
+canvas {
```

Comment:
just curious, is there a way to fix this without adding this CSS? the problem i have with this is that all sketches made before this PR would still be broken, which maybe is inevitable and that's okay, but if there's another way that would be coo.

Reply:
Not that I know of. Without some CSS (`display: block` or `vertical-align: middle`) there is a gap below the canvas because it's inline. It makes it so people can see the whole canvas, but they need to scroll a bit.

It works pretty well without the CSS if you have scrollbars disabled like the default on OSX:

![image](https://user-images.githubusercontent.com/647807/45327401-3f3d0a00-b50c-11e8-8157-a8f006ca43bd.png)

But when scrollbars are visible it gets worse. I don't have a windows machine, so I haven't tried it out, but I suspect Windows always has scrollbars visible 😞 

![image](https://user-images.githubusercontent.com/647807/45327486-87f4c300-b50c-11e8-8b3b-459236df1e96.png)

Reply:
got it! sorry for the delay on this one, but just had a thought—say a user specifically made the height of the canvas larger than the width/height of the preview pane. i think it would make sense for the scrollbars to appear, right? if we included this css, would that make the scrollbars disappear for that case? i feel like it would make sense to keep them in that case?

Reply:
just an update—i've been working on getting a browserstack account (which is free for open source projects) so i can test this.

Reply:
just tested this on windows—scrollbar does appear in the use case when you'd want it to appear.

Reply:
Oh great! Sorry I never got back to this, I couldn't get this to run locally (an issue on my end with some dependency), so I wasn't able to test it again.